public/app/features/plugins/admin/components/PluginDetailsRightPanel.tsx

```diff
@@ -48,7 +48,21 @@ export function PluginDetailsRightPanel(props: Props): React.ReactElement | null
               <Text color="secondary">
                 <Trans i18nKey="plugins.details.labels.updatedAt">Last updated: </Trans>
               </Text>{' '}
-              <Text>{formatDate(new Date(plugin.updatedAt))}</Text>
+              <Text>{formatDate(new Date(plugin.updatedAt), { day: 'numeric', month: 'short', year: 'numeric' })}</Text>
```

Comment:
is this change intentional? it doesn't look like part of this PR scope.

Reply:
Yes, it is, I decided that format 24 Oct 2024 is more readable than 24/10/2024, but this is just me.